pkg/render/intrusion_detection.go

```diff
@@ -309,6 +324,28 @@ func (c *intrusionDetectionComponent) intrusionDetectionClusterRole() *rbacv1.Cl
 				"get", "watch",
 			},
 		},
+		{
+			APIGroups: []string{""},
+			Resources: []string{"podtemplates"},
+			Verbs:     []string{"get"},
```

Comment:
Don't you need create / watch / update as well?

Reply:
we are just using an instantaneous Get for now in IDS controller for now since we will only read values  and create jobs / cronjobs out of them

Reply:
Oh right, these are the permissions for the IDS controller not the operator.